ansible_collections/arista/avd/plugins/README.md

```diff
@@ -176,6 +176,39 @@ To use this filter:
 !!! note
     This isn't using the same range syntax as EOS for modular or break-out ports. For example, on EOS `et1/1-2/4` gives you `et1/1, et1/2, et1/3, et1/4, et2/1, et2/2, et2/3, et2/4` on a fixed switch, but a different result on a modular switch depending on the module types. In AVD, the same range would be `et1-2/1-4`.
 
+#### Password filters
+
+The `arista.avd.encrypt` and `arista.avd.decrypt` filters are used to encrypt or decrypt supported passwords.
+It takes as input a `type`
+
+Both take as parameter a value and key
+
+Supported types:
+
+- bgp
+
+##### BGP passwords
+
+BGP password are encrypted/decrypted based on the Neighbor IP or the BGP Peer Group Name as is expected in EOS.
+
+To use these filters:
```

Comment:
This should go under the general section, since it is not about BGP specifically. The text up there is also not very clear to read.

Reply:
Addressed in 0dbeecf